src/rgw/rgw_user.cc

```diff
@@ -270,10 +270,10 @@ int rgw_get_user_info_from_index(RGWRados *store, string& key, rgw_bucket& bucke
 int rgw_get_user_info_by_uid(RGWRados *store,
                              const rgw_user& uid,
                              RGWUserInfo& info,
-                             RGWObjVersionTracker *objv_tracker,
-                             real_time *pmtime,
-                             rgw_cache_entry_info *cache_info,
-                             map<string, bufferlist> *pattrs)
+                             RGWObjVersionTracker * const objv_tracker,
+                             real_time * const pmtime,
```

Comment:
> Casey noted here, let's not add const to non-pointer/reference arguments.

I can fully agree with that in the matter of *declarations*. However, when it comes to *definitions* `const` for arguments still could be very useful - readers can be sure (of course, with precision to `const_cast<>` or other nasty things) that they are dealing only with labels, not parts of a context.